docs/why/conda-channels-naming-analysis.md

```diff
@@ -0,0 +1,119 @@
+# Conda Channel Naming Discrepancies Analysis Report
+
+## Executive Summary
+
+This report analyzes naming differences between the main conda channel and conda-forge channel for Python packages.
+
+## Data Overview
+
+**72 packages** were found with discrepancies between main conda channel and conda-forge channel names.  This
+was done by collecting information main's packages (from an internal to Anaconda data store) and comparing
+this against conda-forge's cf-graph-countyfair.
```

Comment:
Explaining what or linking to cf-graph-countyfair might help with understanding.